### PR TITLE
feat: author image on feed card

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Experiment to show the author of the post.
- When user is enrolled, do not show the existing hover behavior for displaying the author itself.

Previews:

Screenshot 2024-07-19 at 6 51 25 PM
Screenshot 2024-07-19 at 6 44 30 PM
Screenshot 2024-07-19 at 6 44 18 PM
Screenshot 2024-07-19 at 6 43 58 PM
Screenshot 2024-07-19 at 6 41 15 PM

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

MI-445 #done

### Preview domain
https://mi-445.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/cards/list/PostMetadata.tsx [hunk 1]
```diff
@@ -20,9 +20,9 @@ export default function PostMetadata({
   return (
     <div className={classNames('grid items-center', className)}>
       {topLabel && (
-        <div className="line-clamp-1 font-bold typo-footnote">{topLabel}</div>
```
Comment (sshanzel) [inline packages/shared/src/components/cards/list/PostMetadata.tsx:23]: Wasn't really working.

**Comment by sshanzel** [comment]
[Enrollment approved](https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1721387464907979?thread_ts=1721387420.207379&cid=C02JAUF8HJL).

**Review by rebelchris — APPROVED** [review]
Happy with this :) 
The PostCardHeader is global right, so applied to articles and shared posts + freeform?

**Comment by sshanzel** [comment]
> The PostCardHeader is global right, so applied to articles and shared posts + freeform?

Yes, should be good with that. Just found one issue when viewing it through Squad feed directly, but should be good now :shipit:

**Review by rebelchris — APPROVED** [review]

**Pull request merged**